homeassistant/components/sensor/youless.py

```diff
@@ -0,0 +1,119 @@
+"""
+@ Author      : Gerben Jongerius
+@ Date        : 04/29/2018
+@ Description : Youless Sensor - Monitor power consumption. This component will add the following sensors
+                   - Current power consumption (in W)
+                   - Current tick count, since the Youless meter started running (in W)
+"""
+import json
+import logging
+from datetime import timedelta
+from urllib.request import urlopen
+
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_MONITORED_VARIABLES
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+SENSOR_POWER = 'pwr'
+SENSOR_POWER_NET = 'net'
+SENSOR_P_HIGH = 'p2'
+SENSOR_P_LOW = 'p1'
+SENSOR_GAS = 'gas'
+
+DOMAIN = 'youless'
+CONF_HOST = "host"
```

Comment:
Import this from `const.py`.

Reply:
Thanks for the info on the existing static